cli/hclfmt.go

```diff
@@ -116,3 +125,27 @@ func checkErrors(logger *logrus.Entry, contents []byte, tgHclFile string) error
 	}
 	return nil
 }
+
+func bytesDiff(b1, b2 []byte, path string) (data []byte, err error) {
+	f1, err := ioutil.TempFile("", "")
+	if err != nil {
+		return
+	}
+	defer os.Remove(f1.Name())
+	defer f1.Close()
+	f2, err := ioutil.TempFile("", "")
+	if err != nil {
+		return
+	}
+	defer os.Remove(f2.Name())
+	defer f2.Close()
+	f1.Write(b1)
+	f2.Write(b2)
+	data, err = exec.Command("diff", "--label=old/"+path, "--label=new/"+path, "-u", f1.Name(), f2.Name()).CombinedOutput()
```

Comment:
Not sure if will work "/" path separator in Windows, most probably should be used `filepath.Join()`
Also will be required to have installed `diff` command - worth to mention in documentation, I think